stores/metro.js

```diff
@@ -0,0 +1,116 @@
+const axios = require("axios");
+const utils = require("./utils");
+
+const METRO_WEBSHOP_BASE_URL = "https://shop.metro.at/shop/1";
+const METRO_API_BASE_RL = "https://api.shop.metro.at/2.0";
+const METRO_API_USER = "onlinesortiment11@metro.at";
+const METRO_API_PASSWORD = "Welcome1";
```

Comment:
Woher kommen username und password? Ich fürchte das kann ich aus rechtlichen Gründen nicht mergen.

Reply:
Das sind jene Credentials, welche die [Metro App](https://play.google.com/store/apps/details?id=at.metro.shop.app) verwendet, um alle Produkte zu fetchen (selbst dann wenn man als User noch gar nicht eingeloggt ist).
PW is immer das gleiche, und die E-Mail Adresse bezeichnet eine spezifische Filiale.

Kann prinzipiell jeder, der sich die App anschaut abgreifen, also so pseudo-public würd ich meinen, aber verstehe natürlich deine Bedenken.
Alternativ kann man's vlt als Github-Secret pflegen, wenn's im Code zu heikel is.....